cmd/dp-import-tracker/main.go

```diff
@@ -226,13 +249,18 @@ func main() {
 		logFatal("could not obtain consumer", err, log.Data{"topic": cfg.ObservationsInsertedTopic})
 	}
 
+	dbConnection, err := bolt.NewDriver().OpenNeo(cfg.DatabaseAddress)
```

Comment:
Add a call to dbConnection.close when the app shuts down

Reply:
done 👍